### PR TITLE
Update UI, add animation and memory handling interface

### PR DESCRIPTION
**Pull request by josuemtzmo**
Update UI to a more modern look consistent with Blender > 2.8

WIP:

- [x] Implement animation types:
  - [x] None: Clear image after datacube timesteps finish
  - [x] Extend (default): Continue with last timestep of dataset for as long as animation is played. 
  - [x] Loop: Loop over the dataset to fill the animation.
- [x] Handle memory:
  - [x] Purge methods:
    - [x] Static (default): 
          Only keep n number of frames in memory (default n=5)
    - [x] Dynamic:
          Remove frames depending of memory system.
  - [x] Purge all memory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Learn more about BlenderNC in the official documentation at [https://blendernc.r
 Contributing
 ------------
 
-All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome. More information about contributing to BlenderNC can be found at our [Contribution page](https://blendernc.readthedocs.io/contribute).
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome. More information about contributing to BlenderNC can be found at our [Contribution page](https://blendernc.readthedocs.io/en/latest/contribute.html).
```
Comment (whatnick) [inline README.md:33]: We could use an in repo relative like like [Contribution page](docs/...) as well, [Github](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) also recommends a `contributing.md` which is also an option.

Comment (josuemtzmo) [reply README.md:33]: Could you clarify the contribution page part? I can follow creating a contributing.md, but what do you mean with " repo relative like Contribution page".

Comment (whatnick) [reply README.md:33]: Fixed the typo that had made it incomprehensible.

Comment (josuemtzmo) [reply README.md:33]: Sorry, I still don't get which typo?

**Comment by josuemtzmo** [comment]
Now the default load works, this allows the user to select within "None", "Extend", or "Loop" to continue the animation after the frame is larger than the timesteps of the dataset. 

Screen Shot 2021-07-08 at 5 04 03 pm

**Comment by josuemtzmo** [comment]
Animation and memory handling has been implemented. 
TODO: 

- [ ] Test animation settings.
- [ ] Test memory settings.
- [ ] Check and look for bugs.

**Comment by josuemtzmo** [comment]
- [ ] Add test for language.

**Pull request merged**